guides/ssl-certificates/route53.md

```diff
@@ -0,0 +1,171 @@
+   cert-manager:
+
+    ```console
+    kubectl apply --validate=false -f \
+    https://github.com/jetstack/cert-manager/releases/download/v1.0.1/cert-manager.yaml
+    ```
+
+1. Check that cert-manager installs correctly by running
+
+    ```console
+    kubectl get CustomResourceDefinition | grep cert-manager
+    ```
+
+    You should see certificates, certificate requests, challenges, cluster
+    issuers, issuers, and orders.
+
+1. Next, check that your services are running in the cert-manager namespace
+
+    ```console
+    kubectl get all -n cert-manager
+    ```
+
+## Step 2: Delegate Your Domain Names and Set Up DNS01 Challenges
+
+Because Coder dynamically generates domains (specifically the Dev URLs), your
+certificates need to be approved and challenged. The follow steps will show
+you how to use Route 53 for DNS01 challenges.
+
+1. Log in to AWS Route 53. On the Dashboard, click **Hosted Zone**.
+
+1. Click **Create Hosted Zone**. In the configuration screen, provide the
+**Domain name** that you'll use for Coder (e.g., `coder.exampleCo.com`) and
+make sure that you've selected **Public hosted zone**. Click **Create hosted
+zone** to proceed.
+
+    When your list of hosted zones refreshes, you'll see that your new records
+    includes multiple values under **Value/Route traffic to**.
+
+1. Log in to your DNS provider so that you can edit your NS records.
+
+1. Edit your NS record to delegate your zones to AWS by sending *each* of the
+values under **Value/Route traffic to** to your domain name (i.e., delegate
+`ns-X.awsdns-32.net` to `coder.exampleCo.com`).
+
+## Step 3: Create an IAM Role for `clusterIssuer`
+
+To make sure that your `clusterIssuer` can change your DNS settings, [create the
+required IAM role](https://cert-manager.io/docs/configuration/acme/dns01/route53/#set-up-an-iam-role)
+
+## Step 4: Create the ACME Issuer
+
+1. Using the text editor of your choice, create a new [configuration
+file](https://cert-manager.io/docs/configuration/acme/dns01/) called
+`letsencrypt.yaml` (you can name it whatever you'd like) that includes your
+newly created IAM role:
+
+    ```yaml
+    apiVersion: cert-manager.io/v1
+    kind: ClusterIssuer
+    metadata:
+        name: letsencrypt
+    spec:
+        acme:
+            email: user@example.com
+            preferredChain: ""
+            privateKeySecretRef:
+                name: example-issuer-account-key
+            server: https://acme-staging-v02.api.letsencrypt.org/directory
+            solvers:
+            - dns01:
+                route53:
+                    accessKeyID: your-access-key-ID #secret with IAM Role
+                    region: your-region
+                    secretAccessKeySecretRef:
+                        key: secret-access-key
+                        name: route53-credentials
+            selector:
+                dnsZones:
+                    - yourDomain.com
```

Comment:
```suggestion
              selector:
                  dnsZones:
                      - yourDomain.com
```

Using 4 spaces for indenting is problematic because this "selector" needs to be indented to the same place as the "dns01" which is between solvers and route53. With 4 space indent, the two options are even with solver or event with route53...